src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -942,7 +965,18 @@ impl<'a> QemuCmdLine<'a> {
         self.memory.set_memory_backend_file(&mem_file);
 
         self.machine.set_nvdimm(true);
```

Comment:
Does this still need to be called even if `-machine memory-backend=...` is used?

Reply:
Yeah, then I will put this into the `pci` arm of the match below. Thanks!